docs/pages/inboxes/manage-inboxes.mdx

```diff
@@ -0,0 +1,189 @@
+</div>
+
+You can enable a user to remove a wallet address from their inbox. You cannot remove the recovery wallet address.
+
+## Add a wallet address to an inbox
+
+:::code-group
+
+```jsx [React Native]
+await client.addAccount(walletToAdd)
+```
+
+```kotlin [Kotlin]
+client.addAccount(walletToAdd)
+```
+
+```swift [Swift]
+try await client.addAccount(newAccount: walletToAdd)
+```
+
+:::
+
+## Remove a wallet address from an inbox
+
+:::tip[Note]
+You cannot remove a recovery wallet.
+:::
+
+:::code-group
+
+```jsx [React Native]
+await client.removeAccount(recoveryWallet, addressToRemove)
+```
+
+```kotlin [Kotlin]
+client.removeAccount(recoveryWallet, addressToRemove)
+```
+
+```swift [Swift]
+try await client.removeAccount(recoveryAccount: recoveryWallet, addressToRemove: addressToRemove)
+```
+
+:::
+
+## Revoke all other installations
+
+Your user may encounter a case where a wallet address for one of their installation IDs gets compromised:
+
+<div>
+  <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/0fa5d2dd3c00fb25c0adb7a615408574eb6864a7/docs/pages/img/compromised-address.png" width="350px" />
+</div>
+
+In this case, you can revoke the address’ access to all other installations:
+
+<div>
+  <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/0fa5d2dd3c00fb25c0adb7a615408574eb6864a7/docs/pages/img/revoke-address.png" width="350px" />
+</div>
+
+:::code-group
+
+```jsx [React Native]
+await client.revokeAllOtherInstallations(recoveryWallet)
+```
+
+```kotlin [Kotlin]
+client.revokeAllOtherInstallations(recoveryWallet)
+```
+
+```swift [Swift]
+try await client.revokeAllOtherInstallations(signingKey: recoveryWallet)
+```
+
+:::
+
+## View the inbox state
+
+Find an `inboxId` for an address:
+
+:::code-group
+
+```jsx [React Native]
+const inboxId = await client.inboxIdFromAddress(address)
+```
+
+```kotlin [Kotlin]
+val inboxId = client.inboxIdFromAddress(address)
+```
+
+```swift [Swift]
+let inboxId = try await client.inboxIdFromAddress(address: address)
+```
+
+:::
+
+View the state of any inbox to see the addresses, installations, and other information associated with the `inboxId`.
+
+**Request**
+
+:::code-group
+
+```jsx [React Native]
+const state = await client.inboxState(true)
+const states = await client.inboxStates(true, [inboxId, inboxId])
+```
+
+```kotlin [Kotlin]
+val state = client.inboxState(true) 
+val states = client.inboxStatesForInboxIds(true, listOf(inboxID, inboxID))
+```
+
+```swift [Swift]
+let state = try await client.inboxState(refreshFromNetwork: true)
+let states = try await client.inboxStatesForInboxIds(
+	refreshFromNetwork: true,
+	inboxIds: [inboxID, inboxID]
+)
+```
+:::
+
+**Response**
+
+```json
+InboxState
+{
+	recoveryAddress: string,
+	addresses: string[],
+	installations: string[],
+	inboxId: string,
+}
+```
+
+## FAQ
+
+### What happens when a user removes a wallet address?
+
```

Comment:
I need help describing the anwers here

Reply:
If a user has only one wallet address associated with their inbox that address would be the recovery address and thus it would not be able to be removed. So you cannot remove address from a single wallet account.

If a user has multiple wallets the wallets that get removed no longer have access to the inboxId which means they can no longer be used to be added to groups or dms or read messages from groups or dms they were previously a part of. 
For example say I have 4 addresses associated with my inboxId. A person can start a dm with any of those addresses and they will all be the same dm. Because dms are inboxId based and all 4 off those address are associated with 1 inboxId. However if I remove an address from that inboxId if someone goes to start a new conversation with that address it would not be associated with my inboxId anymore. If I were to log in with that removed address on a new installation it would get a new inboxId but would no longer have access to old groups or conversations.